d2l-typography.html

```diff
@@ -18,6 +18,14 @@
 				@apply(--d2l-font-custom);
 			}
 
```

Comment:
I'd suggest throwing these styles in a mixin (like [the others](https://github.com/Brightspace/d2l-typography-ui/blob/master/d2l-typography-shared-styles.html#L8)] and then just calling it from here. It will allow us to reuse these styles in other things without needing to always apply the CSS class.